app/views/layouts/_nav.html.erb

```diff
@@ -0,0 +1,11 @@
+<div id=“navbar” class=“navbar-collapse collapse”>
+ <ul class=“nav navbar-nav pull-right”>
+   <% if user_signed_in? %>
+     <li><%= link_to "Hello, #{current_user.email}", "#" %></li>
+     <li><%= link_to 'Logout', destroy_user_session_path, method: :delete %></li>
+   <% else %>
+     <li class=“active”> <%= link_to 'Login', new_user_session_path %> </li>
+     <li> <%= link_to 'Sign up', new_user_registration_path %> </li>
+   <% end %>
+ </ul>
+</div>
```

Comment:
I see you are using Bootstrap classes here, but I can’t see that you are actually adding Bootstrap as a dependency. Why?